docs/source/protocols.rst

```diff
@@ -66,18 +66,18 @@ Iteration protocols
 The iteration protocols are useful in many contexts. For example, they allow
 iteration of objects in for loops.
 
-``Iterable[T]``
----------------
+:py:class:`Iterable[T] <typing.Iterable>`
+-----------------------------------------
```

Comment:
Please advise whether the refs in the section titles are useful.

Reply:
I prefer not to have them (consistent with the earlier policy against `code` in titles. But then we need to edit the body text to include a ref, since it's still useful to have it.

Reply:
Maybe a generic line like "See also `ProtocolName` documentation." to each section? The stdlib docs don't shed more light on protocol types TBH; PEP 544 is the reading source if one needs details (and it's not even mentioned in the `typing` documentation!).

Or maybe a line like "The `ProtocolName` type is defined in `typing` module.", just to point out where the type can be imported from?

Reply:
Let's put a line like
```
See also :py:class:`Iterable`.
```
at the end of each section.

Reply:
> (and it's not even mentioned in the `typing` documentation!)

It is in the relevant version (i.e. Python 3.8): https://docs.python.org/3.8/library/typing.html

Reply:
I have removed the remote branch by accident, didn't mean to close this PR.